CHANGELOG.md

```diff
@@ -6,9 +6,10 @@
 
 * `single-port-sg`: Add support for UDP rules
 * `kube-controller-sg`: Update to sync with `single-port-sg` module
+* `ami-centos`: simple module to get the ami id of the specified release of CentOS.
 
 ### Examples
-
+* `ami-centos-test`: New testing example for `ami-centos`.
```

Comment:
Should we just the module in the `vpc-scenario-2` example?

Reply:
I think that depends on the purpose. `vpc-scenario-2` sure can work with this module. But as a test, it involves other modules, not quite a "unit" test.

Reply:
I'd agree with you on that. The one issue I see is that we would need to put more time into maintaining each of the unit tests than we are now.